dev/com.ibm.ws.junit.extensions/src/test/common/junit/runner/OSGiComplianceTestRunner.java

```diff
@@ -61,11 +61,6 @@ public void run(RunNotifier notifier) {
                         numTests++;
                     }
                 }
-            } finally {
-                try {
-                    reader.close();
-                } catch (IOException e) {
-                }
             }
```

Comment:
The previous code swallowed any IOException from the close(); the changed code allows it to propagate.
Did you intend to make this behaviour change ?

Reply:
No. Good spot — I will fix this.

Reply:
Fixed